docs/api-docs/channels/channel-webhooks.md

```diff
@@ -25,6 +25,17 @@ Updates to the following fields trigger a `store/channel/updated` event:
 * `is_enabled` (to be deprecated)
 * `config_meta`
 
+## Channel-aware webhook events
+
+| Name                                                   | Description                                                    |
+|-----------------------------------------------------   |----------------------------------------------------------      |
+| store/channel/{channel_id}/product/assigned            | New product assigned to channel (or new listing creating)      | 

```

Comment:
Product don't need to be new. Those webhooks would trigger when any product is assigned or unassigned to / from channel (via both catalog and listings API).